pkg/blockstorage/azure/azuredisk.go

```diff
@@ -13,9 +13,11 @@ import (
 
 	"github.com/Azure/azure-sdk-for-go/profiles/latest/compute/mgmt/skus"
 	"github.com/Azure/azure-sdk-for-go/profiles/latest/resources/mgmt/subscriptions"
+	azto "github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
 	azcompute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-03-01/compute"
 	"github.com/Azure/azure-sdk-for-go/storage"
-	azto "github.com/Azure/go-autorest/autorest/to"
+	helper "github.com/kanisterio/kanister/pkg/blockstorage"
```

Comment:
```suggestion
```
This is not required. It's already imported on line 24. You can replace `helper.*` to `blockstorage.*`

Reply:
Addressed.